### PR TITLE
add evaluation 02 normal themes

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: code_schemes/kakuma_community_views_on_girls_education_final.json [hunk 1]
```diff
@@ -3,6 +3,62 @@
...
+      "NumericValue": 3,
+      "StringValue": "yes, agree",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-f14b7fe1",
+      "CodeType": "Normal",
+      "DisplayText": "yes, agree but don't practice",
+      "NumericValue": 4,
+      "StringValue": "yes, agree",
```
Comment (IsaackMwenda) [inline code_schemes/kakuma_community_views_on_girls_education_final.json]:
```suggestion
      "StringValue": "yes_agree",
```

Comment (IsaackMwenda) [reply code_schemes/kakuma_community_views_on_girls_education_final.json]: string values are separated with underscores with no spaces.

Comment (IsaackMwenda) [inline code_schemes/kakuma_community_views_on_girls_education_final.json]: this should be 
```suggestion
      "StringValue": ""yes_agree_but_dont_practice"",
```

File: code_schemes/dadaab_community_views_on_girls_education_final.json [hunk 2]
```diff
@@ -3,6 +3,62 @@
...
+      "NumericValue": 3,
+      "StringValue": "yes, agree",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-365c48f4",
+      "CodeType": "Normal",
+      "DisplayText": "yes, girls education is important",
+      "NumericValue": 4,
+      "StringValue": "yes, girls education is important",
```
Comment (IsaackMwenda) [inline code_schemes/dadaab_community_views_on_girls_education_final.json]:
```suggestion
      "StringValue": "yes_girls_education_is_important",
```

**Review by IsaackMwenda — CHANGES_REQUESTED** [review]

File: code_schemes/kakuma_community_views_on_girls_education_final.json [hunk 3]
```diff
@@ -3,6 +3,62 @@
   "Name": "KAKUMA Community Views On Girls Education Final",
   "Version": "0.0.0.2",
   "Codes": [
+    {
+      "CodeID": "code-10590476",
+      "CodeType": "Normal",
+      "DisplayText": "yes",
```
Comment (IsaackMwenda) [inline code_schemes/kakuma_community_views_on_girls_education_final.json]:
```suggestion
      "DisplayText": "yes, no reason",
```

File: code_schemes/kakuma_community_views_on_girls_education_final.json [hunk 4]
```diff
@@ -3,6 +3,62 @@
...
+      "NumericValue": 1,
+      "StringValue": "yes",
```
Comment (IsaackMwenda) [inline code_schemes/kakuma_community_views_on_girls_education_final.json]:
```suggestion
      "StringValue": "yes_no_reason",
```

File: code_schemes/kakuma_community_views_on_girls_education_final.json (hunk 1 above)
Comment (DanielMwendwa) [reply code_schemes/kakuma_community_views_on_girls_education_final.json]: Updated, thanks.

**Review by IsaackMwenda — APPROVED** [review]

**Pull request merged**